docs/user-documentation/moderne-platform/how-to-guides/custom-recipe.md

```diff
@@ -0,0 +1,45 @@
+# Some title
+
+Many people have custom ESLint plugins or other CLI tools that they want to use inside of a recipe so that they can apply it to a large number of repositories.
+
+This guide will walk you through everything you need to know in order to do that.
+
+## Step 1: Create a recipe repository
+
+The first thing you'll need to do is to create a repository for where your recipe should live.
+
+If you are wanting to use a custom ESLint plugin or tool that is available on NPM inside of your recipe, then you should strongly consider forking our [rewrite-codemods repository](https://github.com/moderneinc/rewrite-codemods) as it already includes a lot of the pieces you'll need.
+
+If you are wanting to use some other tool or dependency, consider starting with our [rewrite-recipe-starter repository](https://github.com/moderneinc/rewrite-recipe-starter).
```

Comment:
Should we even include this information in the guide? Or should this be strictly for people that have dependencies in NPM?

Reply:
I think this should probably be just for integrating tools with npm dependencies.